index.js

```diff
@@ -0,0 +1,327 @@
+    });
+}
+
+const reserveTrip = (id) => {
+  reportStatus('');
+  reportStatus('Reserving The Trip...');
+
+  let userData = {
+    'name': $('input[name="user-name"]').val(),
+    'email': $('input[name="email"]').val()
+  }
+  axios.post(URL + `/${id}/reservations`, userData)
+    .then((response) => {
+      reportStatus(`Successfully reserved this trip with the name ${response.data.name}.`);
+      })
+    .catch((error) => {
+      reportStatus(`Encountered an error: ${error.message}.`);
+      });
+
+  $('input[name="user-name"]').val('');
+  $('input[name="email"]').val('');
+}
+
+const showCreateTripForm = () => {
+  reportStatus('');
+  $('#create-trip-form').show();
+  const createTripForm = $('#create-trip-form');
+  createTripForm.empty();
+
+  createTripForm.append(
+    `<h2>Create A Trip</h2>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">Name:</label>
+      <input type="text" name="name" class="trip-form" />
+    </div>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">Continent:</label>
+      <select id="continent" class="trip-form">
+        <option value="" selected disabled>Please select an option...</option>
+        <option value="Africa">Africa</option>
+        <option value="Asia">Asia</option>
+        <option value="Australasia">Australasia</option>
+        <option value="Europe">Europe</option>
+        <option value="North America">North America</option>
+        <option value="South America">South America</option>
+      </select>
+    </div>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">Category:</label>
+      <input type="text" name="category" class="trip-form" />
+    </div>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">Weeks:</label>
+      <input type="number" name="weeks" class="trip-form" />
+    </div>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">Cost($):</label>
+      <input type="number" name="cost" class="trip-form" />
+    </div>`
+  );
+  createTripForm.append(
+    `<div>
+      <label class="trip-form">About:</label>
+      <textarea id="about" name="about" rows="5" cols="20" class="trip-form"></textarea>
+    </div>`
+  );
+  createTripForm.append(
+    `<input type="submit" name="create-trip" value="Create" class="button" id="create-trip"/>`
+  );
+}
+
+const createTrip = () => {
+  reportStatus('');
+  $('#create-trip-form').show();
+  const tripDetails = $('#trip-details');
+  tripDetails.empty();
+  const reservationForm = $('#reservation-form');
+  reservationForm.empty();
+  reportStatus('Creating The Trip...');
+
+  let tripData = {
+    'name': $('input[name="name"]').val(),
+    'continent': $('#continent option:selected').val(),
+    'category': $('input[name="category"]').val(),
+    'weeks': $('input[name="weeks"]').val(),
+    'cost': $('input[name="cost"]').val(),
+    'about': $('#about').val()
```

Comment:
in JavaScript, all properties are assumed to be strings, so you can just write:
```javascript
  let tripData = {
    name: $('input[name="name"]').val(),
    continent: $('#continent option:selected').val(),
    category: $('input[name="category"]').val(),
    weeks: $('input[name="weeks"]').val(),
    cost: $('input[name="cost"]').val(),
    about: $('#about').val()
  }
```
and it should continue to work